pkg/api/openshiftcluster.go

```diff
@@ -151,7 +151,6 @@ type ConsoleProfile struct {
 type ServicePrincipalProfile struct {
 	MissingFields
 
-	TenantID     string       `json:"tenantId,omitempty"`
```

Comment:
Should the `TenantID` field in this other copy of the [`ServicePrincipalProfile` struct](https://github.com/Azure/ARO-RP/blob/ae9fc5d3e295488fda4c03997da8d48308d5cf9e/pkg/api/admin/openshiftcluster.go#L78) be removed too? Might be related to [this e2e failure](https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=37525590&view=logs&j=10f66467-59e5-53bc-0c8a-42cc89770de4&t=3bbe7e73-d181-5aca-4b77-91acc0ef0f4a&s=d654deb9-056d-50a2-1717-90c08683d50a)?

Reply:
Good catch @ihsim!

Reply:
Nice catch, I did this in another branch but after I rebased 😬  🤦🏽 `pkg/api/admin/openshiftcluster.go`